aips/aip-x-disable-object-burn.md

```diff
@@ -0,0 +1,139 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Disable Object Burn
+author: gregnazario (greg@aptoslabs.com)
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Standard (Framework)
+created: 8/28/2024
+updated (*optional):
+requires (*optional):
+---
+
+# AIP-X - Disable Safe Burn of User-Owned Objects
+
+## Summary
+
+Safe burn was originally described in [AIP-45](./aip-45.md), which allowed users to tombstone their objects and remove
+direct linkage of an undeletable, also known as *soulbound*, object to an account. This AIP seeks to disable safe
+object burn, as it caused extra complexity, and sometimes unexpected consequences. Owners of burned objects will still
+be able to unburn and recover their tombstoned objects.
+
+### Goals
+
+As a result of this AIP, users will still be able to unburn their burnt objects, but will not be able to burn any new
+undeletable objects. This will reduce complexity and some interesting edge cases associated.
+
+If we delay enacting this, there may be more burnt objects, which could cause more unexpected behaviors around moving
+burnt objects to the burn address.
+
+### Out of Scope
+
+This removes the previous mitigation, since users cannot opt-out of receiving unsolicited content. Wallets or other
+platforms will need to come up with a standardized way of hiding unwanted items.
+
+Additionally, we will not be preventing unburning existing burnt objects at this time.
+
+## Motivation
```

Comment:
The motivation here is not so clear to me. It mostly mentioned the complexity introduced, but is there a specific example to help understand this complexity better?

Reply:
This leaves room for multiple vulnerabilities, that I did not want to mention.